clinvar/src/main/scala/org/broadinstitute/monster/etl/clinvar/models/output/XRef.scala

```diff
@@ -4,15 +4,15 @@ import io.circe.{Encoder, Json}
 import io.circe.derivation.deriveEncoder
 import upack.{Msg, Str}
 
-case class XRef(db: String, id: String)
+case class XRef(db: String, id: String, sourceName: Option[String])
 
 object XRef {
 
   // Jade doesn't support struct columns yet, so we output these as stringified JSONs.
   implicit val encoder: Encoder[XRef] =
     deriveEncoder.mapJson(obj => Json.fromString(obj.noSpaces))
 
-  def fromRawXRef(rawXref: Msg): XRef = {
+  def fromRawXRef(rawXref: Msg, nameValue: Option[String] = Option.empty): XRef = {
```

Comment:
Can you add scaladoc to describe what `nameValue` is? Also can we give it a more descriptive name like:
```suggestion
  def fromRawXRef(rawXref: Msg, referencedName: Option[String] = Option.empty): XRef = {
```